.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,26 @@
+name: Docker image build
+
+on: [push, release]
+
+jobs:
+  docker-build-push:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
```

Comment:
My understanding is that Qemu is only useful for cross-compiling a multi-platform program, which is not the case here.

Reply:
Yes, I thought cross-compilation might be a requirement. If that is not under plan, then this can be removed.